examples/01-filter/extrude-rotate.py

```diff
@@ -44,3 +45,33 @@
 )
 
 plotter.show(cpos="xy")
+
+###############################################################################
+# Create a spring
+# ~~~~~~~~~~~~~~~
+
+# Create the spring profile (a circle).
+plotter = pyvista.Plotter(shape=(1, 1))
+plotter.subplot(0, 0)
```

Comment:
```suggestion
plotter = pyvista.Plotter()
```
This is already default.

Reply:
Thanks